blackboxopt/optimizers/botorch_base.py

```diff
@@ -277,15 +280,20 @@ def generate_evaluation_specification(self) -> EvaluationSpecification:
 
         if self.num_initial_random > 0 and (
             self.X.size(-2) < self.num_initial_random
```

Comment:
I believe this line is redundant. Both `self.losses` and `self.X` are always of the same size and this line is always satisfied, if the condition below is satisfied. Or am I missing a corner case here?

Reply:
You're right, I'll simplify accordingly.